modified select styling content (#26538)
mdn/content: files/en-us/learn/forms/advanced_form_styling/index.md

}
```

> The second, slightly more major issue is that you don't have control over the box that appears containing the options when you click on the `<select>` box to open it. You can inherit font set on the parent but you won't be able to set things like spacing and colors. The same is also true of the autocomplete list that appears for the datalist.

Reviewer: ```suggestion
The second, slightly more important issue is that you don't have control over the box that appears containing the options when you click on the `<select>` box to open it. You can inherit the font set on the parent, but you won't be able to set things like spacing and colors. The same is true for the autocomplete list that appears with {{HTMLElement("datalist")}}.
```
